scripts/self_update.ts

```diff
@@ -1,6 +1,7 @@
 import Ask from "ask";
 import iro, { bold, yellow } from "iro";
 import { dirname, fromFileUrl } from "std/path";
+
```

Comment:
is there some reason to do this?

Reply:
`TypeScript Hero` extension, which I added to the workspace suggested extensions, adds a line between root imports and relative. I just save all the files to guarantee there will be no changes in the future. Also, I enabled formatting of imports on save.